src/content/en/updates/2017/06/aligning-input-events.md

```diff
@@ -0,0 +1,161 @@
+at. So let's look at a performance timeline from devtools for a simple canvas
+[painting app](https://rbyers.github.io/paint.html).
+
+In the picture below with `requestAnimationFrame()`-aligned input disabled you
+can see multiple processing blocks per frame with an inconsistent frame time.
+The small  yellow blocks indicate hit testing for such things as the target of
+the DOM event, dispatching the event, running javascript, updating the hovered
+node and possibly re-calculating layout and styles.
+
+<figure>
+  <img src="/web/updates/images/2017/06/inconsistent-frame-timing.png"
+       alt="A performance timeline showing inconsistent frame timing."/>
+</figure>
+
+So why are we doing extra work that doesn't cause any visual updates? Ideally we
+don't want to do any work that doesn't ultimately benefit the user. Starting in
+Chrome 60 the input pipeline will delay dispatching continuous events
+([`wheel`](https://developer.mozilla.org/en/docs/Web/Events/wheel),
+[`mousewheel`](https://developer.mozilla.org/en/docs/Web/Events/mousewheel),
+[`touchmove`](https://developer.mozilla.org/en/docs/Web/Events/touchmove),
+[`pointermove`](https://developer.mozilla.org/en/docs/Web/Events/pointermove),
+[`mousemove`](https://developer.mozilla.org/en/docs/Web/Events/mousemove)) and
+dispatch them right before the
+[`requestAnimationFrame()` callback](https://developer.mozilla.org/en-US/docs/Web/API/window/requestAnimationFrame)
+occurs. In the picture below (with the feature enabled) you see a more
+consistent frame time and less time processing events.
+
+We have been running an experiment with this feature enabled on the Canary and
+Dev channels and have found that we perform 35% less hit tests which allows the
+main thread to be ready to run more often.
+
+An important note that web developers should be aware of is that any discrete
+event (such as [`keydown`](https://developer.mozilla.org/en/docs/Web/Events/keydown),
+[`keyup`](https://developer.mozilla.org/en/docs/Web/Events/keyup),
+[`mouseup`](https://developer.mozilla.org/en/docs/Web/Events/mouseup),
+[`mousedown`](https://developer.mozilla.org/en/docs/Web/Events/mousedown),
+[`touchstart`](https://developer.mozilla.org/en/docs/Web/Events/touchstart),
+[`touchend`](https://developer.mozilla.org/en/docs/Web/Events/touchend)) that
+occurs will be dispatched right away along with any pending events, preserving
+the relative ordering. With this feature enabled a lot of the work is
+streamlined into the normal [event loop](https://github.com/atotic/event-loop)
+flow, providing a consistent input interval. This brings continuous events
+inline with [`scroll`](https://developer.mozilla.org/en-US/docs/Web/Events/scroll)
+and [`resize`](https://developer.mozilla.org/en-US/docs/Web/Events/resize) events
+which have already been streamlined into the event loop flow in Chrome.
+
+<figure>
+  <img src="/web/updates/images/2017/06/consistent-frame-timing.png"
```

Comment:
Why don't we see any big purple blocks in this timeline? I would have expected to see fewer, not shorter purple blocks.